tests/performance/ecommerce_hybrid_search/ecommerce_hybrid_search.rb

```diff
@@ -15,8 +15,6 @@ def test_hybrid_search
     dump_thread_stats
 
     benchmark_feed(feed_file_name, "feed")
-    benchmark_queries("after_feed", false, [1])
```

Comment:
Doesn't this mean that we only run un-filtered queries after flush? I think we want both filtered and unfiltered after flush? Or just filtered?

Reply:
All the relevant query benchmarks (both filtered and un-filtered) are found on lines 22 and 23. Lines 18 and 19 are just remains of benchmarking done before flushing, which is not relevant. I remove them to reduce the runtime of the test.